concourse/tasks/daily-statistics.yml

```diff
@@ -0,0 +1,28 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/cf-cli
+    tag: e1ffec0d1940706f157a8c1e0ab8131b7084fa1c
+inputs:
+  - name: git-main
+    path: src
+outputs:
+  - name: statistics
+params:
+  CF_API: https://api.london.cloud.service.gov.uk
+  CF_USERNAME: ((paas-username))
+  CF_PASSWORD: ((paas-password))
+
+run:
+  path: sh
+  args:
+    - "-c"
+    - |
+      set -eu
+
+      cf api "$CF_API"
+      cf auth
+      cf t -o "$CF_ORG" -s "$CF_SPACE"
+
+      cf ssh $CF_APP_NAME -c "/tmp/lifecycle/launcher /home/vcap/app 'rake statistics:general' ''" > statistics/output.json
```

Comment:
This is good for now but I think we'll need to change it when we get around to setting up cyber's ssh alerting.

Reply:
At that point, we should probably review how we get these statistics out, since I imagine we'll still want a dashboard of some type.  Maybe a basic stats endpoint behind some type of authentication would suit our needs?